bot/exts/info/information.py

```diff
@@ -518,12 +518,12 @@ async def json(self, ctx: Context, message: Message) -> None:
         await self.send_raw_content(ctx, message, json=True)
 
     @command(aliases=("rule",))
-    async def rules(self, ctx: Context, rules: Greedy[int]) -> None:
+    async def rules(self, ctx: Context, rules: Greedy[int], keyword: Optional[str]) -> None:
```

Comment:
Because the command currently supports multiple rule integers, I believe it would be intuitive to also support multiple keywords. Ideally, something like `!rules 1 keyword 2 keyword2` would work. However, the current command allows invocations like `!rules 1 2 3 some text here`, and it ignores the "some text here". I'm not sure if this could still be supported given my above proposal.

Need to gather more opinions on what the interface should be.

Reply:
@MarkKoz Yes I agree that it's not intuitive, but it's also not intuitive IMO to invoke it like this `!rules 1 kw1 2 kw2` and also the greedy pattern matching won't work here.

As for when it comes to detecting multiple keywords, we could do that sure, so let's just gather opinions on the interface like you said.

Reply:
Perhaps `!rules 1 kw1 2 kw2` was a poor choice of an example. What I mean is that the keywords and integers should be accepted in any order/grouping. So `kw 1 2 kw 3 kw kw kw 4 5 6 7 kw kw kw 8` would work. Hopefully that's a better example.

Reply:
I'm not sure if that's one of the things that we wanted to do, e.g. drop the priority.
I think that having a pre-known order is also a good thing.

But of course, this just a personal opinion, we could collect more feedback about this to agree on the interface.

Reply:
@MarkKoz After some thought, I figured that it wouldn't hurt to have them in random order & changed the implementation.
Can you have a look now please ?